SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ComponentTestUtils.h

```diff
@@ -240,21 +240,26 @@ inline bool CompareEntityComponentCompleteUpdates(const EntityComponentCompleteU
 	return CompareComponentData(Lhs.CompleteUpdate, Rhs.CompleteUpdate) && CompareComponentUpdateEvents(Lhs.Events, Rhs.Events);
 }
 
-inline bool CompareEntityComponentId(const EntityComponentId& Lhs, const EntityComponentId& Rhs)
+inline bool EntityComponentIdEquality(const EntityComponentId& Lhs, const EntityComponentId& Rhs)
 {
 	return Lhs == Rhs;
 }
 
-inline bool CompareWorkerComponentId(const Worker_ComponentId Lhs, const Worker_ComponentId Rhs)
+inline bool WorkerComponentIdEquality(const Worker_ComponentId Lhs, const Worker_ComponentId Rhs)
 {
 	return Lhs == Rhs;
 }
 
-inline bool CompareWorkerEntityIdKey(const Worker_EntityId Lhs, const Worker_EntityId Rhs)
+inline bool WorkerEntityIdEquality(const Worker_EntityId Lhs, const Worker_EntityId Rhs)
```

Comment:
Possible unnecessery driveby - this is a weird pattern. We could just remove these 3 functions and have an overload to AreEquivalent that doesn't take a predicate.

Reply:
Oh I see (or at least I think I do as I haven't tried to compile it). I think that wasn't done because the functions below are also named AreEquivalent. So That overload can't work. Either those overloads need to be named properly or they should be template specialisations.

Reply:
This can be done in a follow up by whoever though. I just noticed it here.

Reply:
This was just to fix the bug. If you feel strongly about changing this pattern I can look at it as part of this.